ts/Series/PackedBubble/PackedBubbleSeries.ts

```diff
@@ -1347,6 +1383,81 @@ class PackedBubbleSeries extends BubbleSeries implements Highcharts.DragNodesSer
             null;
     }
 
+
+    /**
+     * Set the state of the series. Called internally on mouse interaction
+     * operations, but it can also be called directly to visually
+     * highlight a series.
+     *
+     * @function Highcharts.Series#setState
+     *
+     * @param {Highcharts.SeriesStateValue|""} [state]
+     *        The new state, can be either `'hover'`, `'inactive'`, `'select'`,
+     *        or `''` (an empty string), `'normal'` or `undefined` to set to
+     *        normal state.
+     * @param {boolean} [inherit]
+     *        Determines if state should be inherited by points too.
+     */
+    public setState(
+        state?: (StatesOptionsKey|''),
+        inherit?: boolean
+    ): void {
+
+
+        // Run default method
+        Series.prototype.setState.apply(this, arguments);
+
+        let series = this,
+            options = series.options,
+            stateOptions = options.states,
+            graph = series.graph,
+            layoutOptions = options.layoutAlgorithm,
+            lineWidth,
+            attribs,
+            stateAnimation = pick(
+                (
+                    (stateOptions as any)[state || 'normal'] &&
+                    (stateOptions as any)[state || 'normal'].animation
+                ),
+                series.chart.options.chart.animation
+            ),
+            i = 0;
+
+        if (layoutOptions && layoutOptions.parentNodeOptions && layoutOptions.parentNodeOptions.marker) {
+            lineWidth = (layoutOptions.parentNodeOptions.marker.lineWidth || options.lineWidth);
+
+            if (!series.chart.styledMode) {
+                if (state) {
+                    lineWidth = (
+                        (stateOptions as any)[state].lineWidth ||
+                        lineWidth + (
+                            (stateOptions as any)[state].lineWidthPlus || 0
+                        )
+                    );
+                }
+
+                if (graph && !graph.dashstyle) {
+                    attribs = {
+                        'stroke-width': lineWidth
+                    };
+
+                    // Animate the graph stroke-width.
+                    graph.animate(
+                        attribs,
+                        stateAnimation
+                    );
+                    while ((series as any)['zone-graph-' + i]) {
+                        (series as any)['zone-graph-' + i].animate(
+                            attribs,
+                            stateAnimation
+                        );
+                        i = i + 1;
+                    }
+                }
+            }
+        }
+    }
```

Comment:
This function is largely a repetition of `Series.setState`, which is unfortunate. It also repeats calls to `graph.animate` directly after the super function has called `graph.animate`. 

Is the `lineWidth` the only thing we actually change? In that case we can just change `options.lineWidth` temporarily in the method override. Otherwise, we should consider an event that can handle this more precisely.

```ts
const originalLineWidth = this.options.lineWidth;
this.options.lineWidth = /* insert calculation here */
Series.prototype.setState.apply(this, arguments);
// Reset
this.options.lineWidth = originalLineWidth;
```

Reply:
Thanks for the suggestion! It appeared the entire function was not needed anymore after all.